src/repository/default.rs

```diff
@@ -439,6 +439,54 @@ impl Repository for $repo {
         Ok(functions)
     }
 
+
+    async fn drop_function(
+        &self, database_id:
+        DatabaseId,
+        func_desc: &[sqlparser::ast::DropFunctionDesc],
```

Comment:
A `sqlparser` struct all the way down in our repo implementation feels out of place. We should probably parse the required info during logical planning and populate adequate fields in our extension node at that point.

On that note, given that the function name is a unique key in our case (in [combination with the DB id](https://github.com/splitgraph/seafowl/blob/main/migrations/postgres/20220705072248_create_tables.up.sql#L68)), there's no need to require/keep track of the args, only the names supplied in the statement.

Reply:
Thanks for the pointer to the DB id definition. 
Given the function names provide sufficient uniqueness it seems like `func_names: Vec<String>` should be safe. I went ahead and did this in the most recent commit. 
Going to mark resolved, but if I've made an error please feel free to un-resolve/comment as needed.